ClientServer.go

```diff
@@ -33,7 +32,7 @@ func Server() {
 		// will listen for messages from client
 		message, err := bufio.NewReader(conn).ReadString('\n')
```

Comment:
bufio.NewReader(conn) reads data as string object and returns (msg string, err error) while conn.Read(b) reads data as byte object and returns (n int , err error) . Here 'n' is the length of bytes and moreover we have to pass a fixed length byte object into the "conn.Read(b)" means we define length for the data that conn.Read(b) can read.After that we use variable 'n' to make string. While bufio.NewReader(conn) simply returns a string for whatever it reads.